app/bundles/LeadBundle/Form/Type/DashboardSegmentContactsWidgetType.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Mautic\LeadBundle\Form\Type;
```

Comment:
All new classes should declare strict types
```suggestion
<?php

declare(strict_types=1);

namespace Mautic\LeadBundle\Form\Type;
```